File: hw/jacome/U3/HW19AbstractFactory/JS/view/LinuxButton.js
const Button = require('../model/Button');

class LinuxButton extends Button {
    paint() {
        console.log("I'm a LinuxButton: " + this.caption);
        console.log("------------");
        console.log("| " + this.caption.padEnd(11) + " |");
        console.log("------------");
    }
}

module.exports = LinuxButton;

File: hw/sivinta/U3/HW18GUIFactory/GUIFactory/src/main/java/ec/edu/espe/guifactory/view/LinuxButton.java
package ec.edu.espe.guifactory.view;

import ec.edu.espe.guifactory.model.Button;


/**
 *
 * @author Jahir Sivinta DDCO
 */
public class LinuxButton extends Button {

    public void paint() {
        System.out.println("I'm a LinuxButton: " + caption);
    }
}
